prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -6029,11 +6029,16 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
         match op {
             mir::NullOp::SizeOf => {
                 // TODO: ParamEnv::empty() should probably be tyctxt.param_env(def_id_of_method)
```

Comment:
Actually since we are changing this, does it help to apply the suggestion in this TODO, i.e. to use `self.encoder.env().query.tcx().param_env(self.proc_def_id)` instead of `ParamEnv::empty()`?

Reply:
Addressed in https://github.com/viperproject/prusti-dev/pull/1246/commits/6c21fc1adc9d798ab4d50d6128deddcb42bb4f98

Reply:
Unfortunately it doesn't prevent getting the `Err`, but yeah might as well

Reply:
@frewsxcv Could you please say if the change fixes the original issue in the `geo-types` crate? I am hoping that an MIR `SizeOf` is only emitted by `rustc` when the layout can be determined.